java/code/src/com/suse/manager/reactor/messaging/RefreshGeneratedSaltFilesEventMessageAction.java

```diff
@@ -148,6 +157,8 @@ public void refreshFiles() throws IOException {
             if (Files.exists(tempCustomPath)) {
                 // this condition is needed only at setup time when there are no orgs yet
                 Files.move(tempCustomPath, saltPath, StandardCopyOption.ATOMIC_MOVE);
+                setAttributes(saltPath, "tomcat", "susemanager",
+                        Set.of(OWNER_READ, OWNER_WRITE, GROUP_READ, GROUP_WRITE));
```

Comment:
saltPath is a directory. Do we need execute permissions?

Reply:
And do we need group write permissions? 

  Set.of(OWNER_READ, OWNER_WRITE, OWNER_EXECUTE, GROUP_READ, GROUP_EXECUTE)

Reply:
right i changed it to match the other place where it changes directory permissions

Reply:
i think this statement might be redundant anyway since it does not create the directory but goes though `generateStateAssignmentFile` which creates it and sets permissions already.